src/main/java/org/terasology/scenario/internal/events/ConvertIntoEntityConstantEvent.java

```diff
@@ -17,6 +17,11 @@
 
 import org.terasology.entitySystem.event.Event;
 
+/**
+ * Event that for converting a single serialised string(generated by ConvertEntitySystem) back into an entity.

```

Comment:
With {@link className} you can create links in javadoc to classes of which the IDEs will then be aware of when you rename.